index.js

```diff
@@ -1,7 +1,5 @@
-const log = value => {
-  if (process.env.NODE_ENV !== 'test') {
-    console.log(value);
+module.exports = new Proxy( { }, {
```

Comment:
Instead of passing an object as target, I think if you pass the console global here it would make it easier to understand and also when you inspect the resulting object of this module you would see it's a proxy of console, so it gives a better experience for the user, what do you think?

Reply:
:+1: